boards/arm/nrf9161dk_nrf9161/nrf9161dk_nrf9161.yaml

```diff
@@ -6,7 +6,7 @@ toolchain:
   - gnuarmemb
   - xtools
   - zephyr
-ram: 88
+ram: 256
 flash: 1024
```

Comment:
Scratch partition should be removed from 9161 also (can do it in a different PR)

Reply:
just added the change for the nrf9160dk as well :+1: